src/Patches/TunicPortals.cs

```diff
@@ -943,5 +979,261 @@ public static void AltModifyPortals(Dictionary<string, PortalCombo> portalComboL
                 }
             }
         }
+
+
+        public static string PythonOutput()
```

Comment:
I'm assuming this was just to help you with AP stuff, and isn't needed otherwise?

Reply:
Yup, not needed otherwise. Was debating just saving it elsewhere then deleting it from here (cause if I have to majorly change it I ain't writing this all out again lol)